core/src/main/java/org/apache/accumulo/core/rpc/ThriftUtil.java

```diff
@@ -63,12 +66,16 @@ public class ThriftUtil {
   private static final int RELOGIN_MAX_BACKOFF = 5000;
 
   /**
-   * An instance of {@link TraceProtocolFactory}
+   * An instance of {@link AccumuloProtocolFactory}
    *
    * @return The default Thrift TProtocolFactory for RPC
    */
-  public static TProtocolFactory protocolFactory() {
-    return protocolFactory;
+  public static TProtocolFactory clientProtocolFactory() {
+    return clientAccumuloProtocolFactory;
+  }
+
+  public static TProtocolFactory serverProtocolFactory() {
+    return serverAccumuloProtocolFactory;
```

Comment:
This @ return may need to be updated. Should also probably have a similar javadoc for `serverProtocolFactory()`